Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.6 as step1
+COPY . /app
+RUN pip install --upgrade pip
+RUN pip install -r ./requirements.txt
+ENV ENVIRONMENT production
+CMD ["python","/data_process/tsv_to_dataframe.py"]
```

Comment:
this CMD doesn't seem to work, it is not finding the data_set files when ran from Docker I believe it has to do with tsv_to_dataframe.py expecting to be located in the folder where it runs. 
Not sure tho if it is best to fix this on the script itself @brunomichetti , or work around this in the Dockefile